io/src/lib.rs

```diff
@@ -0,0 +1,184 @@
+#![no_std]
+
+use codec::{Decode, Encode};
+use gstd::{prelude::*, ActorId};
+use scale_info::TypeInfo;
+
+#[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Clone)]
+pub enum Move {
+    Rock,
+    Paper,
+    Scissors,
+    Lizard,
+    Spock,
+}
+
+impl Move {
+    pub fn number(&self) -> char {
```

Comment:
Need to consider whether we can use `u8` and byte slices instead of wide char and strings. It will be a little bit more efficient I guess.